testing/content/files/en-us/web/brokenlinks/index.html

```diff
@@ -26,3 +26,5 @@
 <p><a href='/en-US/docs/Hopeless/Case'>Will not have a suggestion</></p>
 
 <p><a href="//www.peterbe.com">Leave me alone! I'm actually external</a></p>
+
+<p><a href="/en-US/docs/Web/BrokenLinks">Link to itself</a></p>
```

Comment:
We should add something that ensures that links to the same page that include hashtag's are not considered broken links.

Reply:
Excellent idea! In fact, we should add 2 more:
```html
<p><a href="#anchor">Link to itself</a></p>
<p><a href="/en-US/docs/Web/BrokenLinks#anchor">Link to itself</a></p>
```
where in the second example, people needlessly type in the path and not just the anchor. I've seen this more than once.